Formula/judy.rb

```diff
@@ -0,0 +1,80 @@
+class Judy < Formula
+  desc "State-of-the-art C library that implements a sparse dynamic array"
+  homepage "https://judy.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/judy/judy/Judy-1.0.5/Judy-1.0.5.tar.gz"
+  sha256 "d2704089f85fdb6f2cd7e77be21170ced4b4375c03ef1ad4cf1075bd414a63eb"
+  license "LGPL-2.1-or-later"
+
+  # Fix -flat_namespace being used on Big Sur and later.
+  patch :DATA
```

Comment:
Do none of our usual `-flat_namespace` patches apply?

Reply:
the configure is too old for the patch

Reply:
But the patch seems pretty straightforward? I can go either way though.

Reply:
Seems nicer to regenerate `configure`, especially if it's very old. Who knows what else is outdated about it.